annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -202,6 +207,7 @@ protected void validateVcardAddressBody(Body body) throws ParamValidationExcepti
      */
     protected void validateSemanticTagVcardAddressBody(Body body)
 	    throws ParamValidationException, PropertyValidationException {
+    if(body==null) return;
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown

Reply:
Improved.